ai-feature-pack/src/main/resources/layers/standalone/mcp-server/layer-spec.xml

```diff
@@ -10,6 +10,7 @@
     </props>
     <dependencies>
         <layer name="mcp"/>
+        <layer name="elytron-oidc-client"/>
```

Comment:
Why this dependency is not in secured-mcp-server?

Reply:
because secured-mcp-server depends on mcp-server